src/_articles/language/mixins.md

```diff
@@ -13,17 +13,20 @@ Looking for an introduction to mixins?
 See the Language Tour section [Adding features to a class: mixins][].
 
 This article provides a brief theoretical presentation of the evolution of mixins in Dart.
-Support for mixins changed in Dart 1.13, and again in Dart 2:
+Support for mixins changed in Dart 1.13, and it will again in Dart 2:
 
 - Dart 1.12 or lower supports mixins that must extend `Object`, and must not
   call `super()`.
-- Dart 1.13 or greater (< Dart 2) supports mixins that can extend from classes
+- Dart 1.13 or greater supports mixins that can extend from classes
   other than `Object`, and can call `super.method()`. This support is only
   available by default in the Dart VM and in Analyzer behind a flag. More
   specifically, it is behind the `--supermixin` flag in the command-line
   analyzer. It is also available in the analysis server, behind a
   client-configurable option. Dart2js and dartdevc do not support super mixins.
-- Dart 2 deprecates `--supermixin`.
+- In Dart 2, mixins are expected to have fewer restrictions. For example,
+  Flutter supports mixins calling `super()` and extending from a class other
+  than `Object`, but the syntax is expected to change before appearing in all
+  Dart SDKs. For details, see [Dart 2.0 Mixins.][Dart 2.0 Mixins]
```

Comment:
Let's change the name of the link, since the feature won't be in the initial release (2.0). How about:

For details, see the [proposed mixin specification.][...]